packages/core/src/document.ts

```diff
@@ -124,13 +123,6 @@ class Document extends EventEmitter {
 
     const record = await dispatcher.retrieveRecord(doc._genesisCid)
 
-    let payload
-    if (DoctypeUtils.isSignedRecord(record)) {
-      payload = await dispatcher.retrieveRecord(record.link)
-    } else {
-      payload = record
-    }
```

Comment:
Why is this no longer needed?

Reply:
I guess it has been refactored before since payload is not being used anymore.